modules/compiler/targets/riscv/test/lit/passes/link-builtins-memmove.ll

```diff
@@ -0,0 +1,52 @@
+; Copyright (C) Codeplay Software Limited
+;
+; Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+; Exceptions; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+;
+;     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+; WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+; License for the specific language governing permissions and limitations
+; under the License.
+;
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; RUN: muxc --device "%riscv_device" --passes link-builtins,verify -S %s | FileCheck %s
+
+target triple = "riscv64-unknown-unknown-elf"
+target datalayout = "e-p:64:64:64-m:e-i64:64-f80:128-n8:16:32:64-S128"
+
+; CHECK-NOT: call void @llvm.memcpy
```

Comment:
Ideally, we'd check there that *either* no call to `memcpy` is emitted, *or* a definition of `memcpy` is emitted. But that isn't something we can easily do in FileCheck, so this test is fine as is.

Reply:
agreed, nature of lit tests.